### PR TITLE
Moved reaction_test and reaction_latex inside CS

### PR DESCRIPTION
**Pull request by gracetangg**
This pull request tackles moving reaction_latex function inside the CS class #24. I used a lot of what Nancy had wrote before, but I finished adding `reaction_latex` to the CS class. I also changed the function calls from the test files and from the `__init__.py` files. 

Right now `reaction_text_side` and `reaction_latex_side` are very similar, so I was thinking about working on a way to combine those two too, so we only have one function that both `reaction_latex` and `reaction_text` can call

### CODE REVIEW

**Comment by padamson** [comment]
This looks great, and I like the idea of refactoring to just reaction_latex and reaction_text. Could you make a test for these? (Follow red->green->refactor).

**Comment by gracetangg** [comment]
I created some tests for the reaction_text_side changes, but I'm not sure if the tests I created for reaction_latex are too in depth. If it works I can also do the same for the reaction_text tests so they are more exhaustive.

File: nepc/nepc.py [hunk 1]
```diff
@@ -387,11 +384,122 @@ def __init__(self, cursor, cs_id):
...
+            'LHS' or 'RHS' to indicate which side of the reaction to return.
+        latex : bool, default False
+            Determine whether or not to output the LaTeX formatted text. 
+        Returns
+        -------
+        : str
+            The plain text for either the LHS or RHS of the process involved in a NEPC cross section
+            (LaTeX) formatted if indicated.
+        """
+        # FIXME: move this method to the CS Class
```
Comment (padamson) [inline nepc/nepc.py:417]: Can delete this comment
```suggestion
```

**Comment by padamson** [comment]
I really like the exhaustive tests for `reaction_latex`, and it would be good to have those for the plain text version.

**Comment by padamson** [comment]
It looks like instance attributes are being set when the function is called. I wonder if [lazily-evaluated property pattern](https://github.com/faif/python-patterns/blob/master/patterns/creational/lazy_evaluation.py#L19) would be good here.

**Comment by gracetangg** [comment]
@padamson I added a `lazy_evaluation` property to nepc so `reaction_latex` and `reaction_text` are evaluated as attributes. I didn't know where I should include the `lazy_evaluation` definition though (if it should be inside the CS class for organization or if it works outside of the class where I have placed it)

**Comment by padamson** [comment]
@gracetangg I kind of like the definition of the `lazy_evaluation` property being outside of the `CS` class since we probably want to use it elsewhere. Is this ready to merge?

**Pull request merged**